src/main/helm/helm-service/list-helm-charts.injectable.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import assert from "assert";
+import { object } from "../../../common/utils";
+import { HelmChartManager } from "../helm-chart-manager";
+import getActiveHelmRepositoriesInjectable from "../repositories/get-active-helm-repositories/get-active-helm-repositories.injectable";
+
+const listHelmChartsInjectable = getInjectable({
+  id: "list-helm-charts",
+
+  instantiate: (di) => {
+    const getActiveHelmRepositories = di.inject(getActiveHelmRepositoriesInjectable);
+
+    return async () => {
+      const result = await getActiveHelmRepositories();
+
+      assert(result.callWasSuccessful);
```

Comment:
why not return `[]`? Or is this match previous behaviour?

Reply:
Old implementation just extracted to own file since we had to modify couple of them.